src/components/NavItems/Assistant/AssistantScrapeResults/AssistantCredibilitySignals.jsx

```diff
@@ -265,6 +265,7 @@ const AssistantCredSignals = () => {
           onChange={handleChange(newsFramingTitle)}
           disabled={newsFramingLoading || newsFramingFail}
           disableGutters
+          data-testid="topic-accordion"
```

Comment:
These will be changing from Accordion to CustomTabPanel for genre, topic, persuasion and subjectivity. https://github.com/AFP-Medialab/verification-plugin/blob/deeec8edc2f7a2496ded30035ae647f6027468a7/src/components/NavItems/Assistant/AssistantScrapeResults/AssistantTextResult.jsx#L244